django/db/backends/sqlite3/operations.py

```diff
@@ -76,6 +76,13 @@ def date_extract_sql(self, lookup_type, field_name):
         """
         return "django_date_extract('%s', %s)" % (lookup_type.lower(), field_name)
 
+    def fetch_returned_insert_rows(self, cursor):
+        """
+        Given a cursor object that has just performed an INSERT...RETURNING
+        statement into a table, return the tuple of returned data.
```

Comment:
`fetchall()` returns a list not a tuple on SQLite.
```suggestion
        statement into a table, return the list of returned data.
```